fabric/build.gradle

```diff
@@ -25,11 +25,19 @@ repositories {
     maven {
         url "https://jitpack.io"
     }
+    maven {
+        url "https://maven.terraformersmc.com/"
+    }
 }
 
 dependencies {
     modImplementation "net.fabricmc:fabric-loader:${fabric_loader_version}"
     include(modImplementation(fabricApi.module("fabric-resource-loader-v0", fabric_version)))
+    include(modImplementation(fabricApi.module("fabric-screen-api-v1", fabric_version)))
+    include(modImplementation(fabricApi.module("fabric-key-binding-api-v1", fabric_version)))
+    include(modImplementation(fabricApi.module("fabric-lifecycle-events-v1", fabric_version)))
```

Comment:
Are these actually needed? From what I understand, modmenu is optional. _If modmenu is present_, then these become dependencies of modmenu, not us.

Reply:
I think you are correct. All that is needed for ModMenu is to provide a proper implementation of its `ModMenuApi` class.